### DISCUSSION TITLE
Make Explod Parameters Trackable via Trigger

### DISCUSSION THREAD

**Discussion by Kokorz** [ikemen-engine/Ikemen-GO#1453] [Feature Request]
### Is your feature request related to a problem? Please describe.

So, I'm making onomatopoeia effects and someone had the idea of increasing the scale based on the length of the hitpause and making the explod loop certain frames.  You'd think animelemlooped would help, however, that only loops the current animelem the explod is on, rather than allowing you to set a separate loop point. Additionally, I would prefer not to create additional animations just to try and get this functionality to work, as this would mean editing a mass amount of explods.

### Describe the solution you'd like

Make all explod parameters trackable via a trigger called `explod`/`explod(x)`. Similar to the likes of `helper(x), trigger` or `gethitvar(x)`, this allows you to see what parameters the explod has right now, be it scale, its anim, its angle, the animelem it started at, so on so forth. Along with that, a few more parameters would be great. such as `currentanimelem` for checking the current anim elem of the explod, `currentanimticks` for checking the current animation tick the explod is on, and `time` for checking how long the explod has even existed.

### Search terms

explod trigger track parameter

**Comment by potsmugen** [comment]
This sounds like a bit much. You can already use helpers if you want more control.
